src/baldr/graphtile.cc

```diff
@@ -571,6 +571,19 @@ std::vector<SignInfo> GraphTile::GetSigns(const uint32_t idx) const {
   return signs;
 }
 
+// TODO: THIS SHOULD BE MOVED INTO A UTILITY
+uint64_t GraphTile::GetSpeed(const DirectedEdge* de) const {
```

Comment:
could be very simple: return (de->free_flow_speed() > 0) ? de->free_flow_speed() : de_speed();
Though later on it may need to be more complicated. Would be good for this to be inline (only in the .h file) as well as this will be called a lot within the route path algorithms.